### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by leyuan**
Create PR template

https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### What Did I do
+
+### Before
+
+### After
+
+### Reference
```
Comment (AaronLiuIsCool) [inline .github/PULL_REQUEST_TEMPLATE.md]: Jira item number or Trello task item tracking?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### What Did I do
+
+### Before
+
+### After
+
```
Comment (AaronLiuIsCool) [inline .github/PULL_REQUEST_TEMPLATE.md]: Unit test status. check mark

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (leyuan) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can be anything

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (leyuan) [inline .github/PULL_REQUEST_TEMPLATE.md]: should our test be hooked up with CircleCI or something, I will add for now!

**Comment by leyuan** [comment]
@AaronLiuIsCool added

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (AaronLiuIsCool) [inline .github/PULL_REQUEST_TEMPLATE.md]: Good call!

**Comment by AaronLiuIsCool** [comment]
LGTM.

**Pull request merged**